src/cmd/kssh/kssh_test.go

```diff
@@ -46,7 +46,7 @@ func TestIsValidCert(t *testing.T) {
 }
 
 func BenchmarkLoadConfigs(b *testing.B) {
-	os.Remove("~/.ssh/kssh.config")
+	os.Remove("~/.ssh/kssh-config.json")
```

Comment:
NAB: it's a little sloppy that this file is named in so many different places. it would be nicer if it were in just one place and pulled in everywhere it's needed.

Reply:
Yeah, I definitely agree. I've tried to keep it to a minimum but the split between the go codebase and the python integration tests make it difficult (at least without adding some avdl etc).